src/components/Collection.tsx

```diff
@@ -1,5 +1,4 @@
-/* @flow */
-import * as React from 'react';
+import React from 'react';
```

Comment:
Let's keep it following everywhere

```suggestion
import * as React from 'react';
```